Conversions/DateToDay.js

```diff
@@ -9,34 +9,34 @@
     algorithm shown below gives us the number of the day and
     finally converts it to the name of the day.
 
-    Algorithm & Explanation : https://cs.uwaterloo.ca/~alopez-o/math-faq/node73.html
+    Algorithm & Explanation : https://en.wikipedia.org/wiki/Zeller%27s_congruence
 */
 
 // March is taken as the first month of the year.
 const calcMonthList = {
```

Comment:
This mapping should (1) use an array and (2) definitely needs a comment; "March is taken as first month of the year" explains nothing. I prefer the PRs that get rid of it entirely.

Reply:
Noted.
Updated code as per comment.

I had thought of making it array as you suggested, like below:

```js
/**
 * This array holds months number
 * Considering March as the first month of the year
 * For January & February we store value as 13 & 14 respectively,
 * we consider these two from last year
 * And for other months we store exact month number
 */
const monthArray = [13, 14, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
```

/# And then use it as **monthArray[month - 1]**
```js
const weekDay = Math.abs((day + Math.floor((monthArray[month - 1] + 1) * 2.6) + yearDigit + (yearDigit / 4) + (century / 4) - (2 * century)) % 7)
```

Then updated the code and remove this dependency completely.